.gitignore

```
@@ -1,2 +1,3 @@
 /machine-approver
 .idea*
+bin
```

I think you should make this a folder, else it will include any `bin` anywhere

Also, new lines at the end of files please
```suggestion
bin/

```